src/components/Header.jsx

```diff
@@ -0,0 +1,27 @@
+import { Image, Button } from "@chakra-ui/react";
+import myImg from "../assets/jiaksimi1.png";
+import { useState } from "react";
+import { Link } from "react-router-dom";
+
+export default function Header() {
+  const [show, setShow] = useState(true);
+
+  const handleClick = () => {
```

Comment:
This handleClick does button toggling, but you have already wrapped it in a Link, so this may be redundant